meson.build

```diff
@@ -0,0 +1,76 @@
+project('libebml', 'cpp',
+  version : '1.4.2',
+  default_options : ['warning_level=1', 'cpp_std=c++14'],
+)
+
```

Comment:
It's missing the `ENABLE_WIN32_IO` compilation option.

Also missing the `HAVE_WINAPIFAMILY_H` definition.

Reply:
Added both.